net-wireless/nitroshare/nitroshare-0.4.0~git.recipe

```diff
@@ -60,15 +66,20 @@ defineDebugInfoPackage nitroshare$secondaryArchSuffix \
 
 BUILD()
 {
-	cmake -DCMAKE_INSTALL_PREFIX:PATH=$appsDir .
+	cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX:PATH=$appsDir .
```

Comment:
I haven't checked this, does it use gnuInstallDirs? (if so you could use $cmakeDirAgrs and -DCMAKE_INSTALL_BINDIR=$appsDir)

Reply:
I'm utilizing this way for now. May review further when we create icon for desktop menu.